Cargo.toml

```diff
@@ -279,6 +279,7 @@ stm32f446 = ["stm32f4/stm32f446", "device-selected", "gpio-f446",
     "otg-fs",
     "otg-hs",
     "sai",
+    #"sdio",
```

Comment:
What's up with that?

Reply:
446 has sdio, but its svd has several issues